extensions/binding/org.eclipse.smarthome.binding.npt/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,24 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: npt Binding
+Bundle-SymbolicName: org.eclipse.smarthome.binding.npt;singleton:=true

```

Comment:
Crap... I must have made this error at the very first creation of the bundle. Will fix this (incl all the other references to npt)